README.md

```diff
@@ -162,5 +162,5 @@ function handleSquirrelEvent() {
 You can get debug messages from this package by running with the environment variable `DEBUG=electron-windows-installer:main` e.g.
 
 ```
-DEBUG=electron-windows-installer:main node tasks/electron-winstaller.js
+DEBUG=electron-windows-installer node tasks/electron-winstaller.js
```

Comment:
If you want to debug — you need all categories in any case. But if you use different categories, you cannot get time information (debug measures time per category).